### PR TITLE
Experiments example using python sdk

### PR DESCRIPTION
**Pull request by dvasani-CS**

### CODE REVIEW

File: ExperimentsExample/Makefile [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+predict: build-predict push-predict deploy-predict tests-predict
+
+all: types train predict
+
+get: check-env
+	cortex skills describe ${SKILL_NAME} --project ${PROJECT_NAME}
+
+check-env:
+ifndef DOCKER_PREGISTRY_URL
```
Comment (yreddy-CS) [inline ExperimentsExample/Makefile:49]: From the upcoming release, if the user is using the private registry provided by then the URl can be obtained from url <baseURl>/fabric/v4/info(Eg: https://api.dci-dev.dev-eks.insights.ai/fabric/v4/info ) which has the private registry under the endpoints key
If this comment is being addressed then Readme has to be updated too

File: ExperimentsExample/Makefile [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+	$(error environment variable DOCKER_PREGISTRY_URL is not set. Set it like <docker-registry-url>/<namespace-org>)
+endif
+ifndef PROJECT_NAME
+	$(error environment variable PROJECT_NAME is not set. Set this to Cortex project name.)
```
Comment (yreddy-CS) [inline ExperimentsExample/Makefile:53]: If the user has set a default project in the cortex profile, we can pick that too.
If this comment is being addressed then Readme has to be updated too

**Review by yreddy-CS — APPROVED** [review]

**Review by jlara-cs — APPROVED** [review]
lgtm

File: ExperimentsExample/predict/Dockerfile [hunk 3]
```diff
@@ -0,0 +1,5 @@
+FROM python:3.7.4-slim
```
Comment (jgielstra-cs) [inline ExperimentsExample/predict/Dockerfile:1]: Bump to a newer python version ? 3.9.5 ?

**Review by jgielstra-cs — APPROVED** [review]

**Pull request merged**